/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f415e70f350>: Failed to establish a new connection: [Errno 111] Connection refused',))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m...................................................................................
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m...........................................................Exception: ('Connection aborted.', error(104, 'Connection reset by peer'))
[92m Sent! [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 87, in deploy_contract
    MyGlobals.web3.personal.unlockAccount(etherbase, '1', 15000)
  File "/usr/local/lib/python2.7/dist-packages/web3/personal.py", line 93, in unlockAccount
    [account, passphrase, duration],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7a67d13450>: Failed to establish a new connection: [Errno 111] Connection refused',))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', error(104, 'Connection reset by peer'))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m............................................................
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[91m[-] Some of the files is missing or empty: |AccountModifiersInterface.abi|=533  |AccountModifiersInterface.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m...................................................Exception: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5f0554f450>: Failed to establish a new connection: [Errno 111] Connection refused',))
[92m Sent! [0m
[91m[-] Some of the files is missing or empty: |AccountModifiersInterface.abi|=533  |AccountModifiersInterface.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[91m[-] Some of the files is missing or empty: |AccountModifiersInterface.abi|=533  |AccountModifiersInterface.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[91m[-] Some of the files is missing or empty: |Token.abi|=2156  |Token.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m.............................................................Exception: ('Connection aborted.', error(104, 'Connection reset by peer'))
[92m Sent! [0m
[91m[-] Some of the files is missing or empty: |Token.abi|=2156  |Token.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[91m[-] Some of the files is missing or empty: |Token.abi|=2156  |Token.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[91m[-] Some of the files is missing or empty: |TradeTrackerInterface.abi|=434  |TradeTrackerInterface.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m.........................................................................................Exception: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fca6c6df450>: Failed to establish a new connection: [Errno 111] Connection refused',))
[92m Sent! [0m
[91m[-] Some of the files is missing or empty: |TradeTrackerInterface.abi|=434  |TradeTrackerInterface.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[91m[-] Some of the files is missing or empty: |TradeTrackerInterface.abi|=434  |TradeTrackerInterface.bin|=0
The contracts have NOT been deployed[0m
[91m[-] Cannot deploy the contract [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_obj_to_text` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_text` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', error(104, 'Connection reset by peer'))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_obj_to_text` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_text` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m......................................................................................................................................................
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:167: DeprecationWarning: The encoding keyword has been deprecated.  Please update your code to use sha3(text='txt'), sha3(hexstr='0x747874'), sha3(b'\x74\x78\x74'), or sha3(0x747874).
  "The encoding keyword has been deprecated.  Please update your "
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_obj_to_bytes` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/eth_utils/formatting.py:20: DeprecationWarning: The `is_prefixed` function has been deprecated and will be removed in a subsequent release of the eth-utils library. Now that eth-utils is compatible with Python3, we encourage developers to use stdlib functions where possible.
  "functions where possible.".format(fn.__name__)
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m........................................................Exception: ('Connection aborted.', error(104, 'Connection reset by peer'))
[92m Sent! [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 87, in deploy_contract
    MyGlobals.web3.personal.unlockAccount(etherbase, '1', 15000)
  File "/usr/local/lib/python2.7/dist-packages/web3/personal.py", line 93, in unlockAccount
    [account, passphrase, duration],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f484c7415d0>: Failed to establish a new connection: [Errno 111] Connection refused',))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_obj_to_text` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/eth_utils/string.py:22: DeprecationWarning: The `force_text` function has been deprecated and will be removed in a subsequent release of the eth-utils library. UTF8 cannot encode some byte values in the 0-255 range which makes naive coersion between bytes and text representations impossible without explicitly declared encodings.
  "declared encodings.".format(fn.__name__)
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', BadStatusLine("''",))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m......................................................................
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Deploying contract [0m...........................................................................................................[92m confirmed at address: 0x9E536236ABF2288a7864C6A1AfaA4Cb98D464306 [0m
[1m[ ] Contract code length on the blockchain : [0m [92m  7812  : 0x6080604052600436106100625760... [0m
[1m[ ] Contract address saved in file: [0m[92m./out/InstantTrade.address [0m
[94m[ ] Check if contract is SUICIDAL[0m

[ ] Contract address   : 0x9E536236ABF2288a7864C6A1AfaA4Cb98D464306
[ ] Contract bytecode  : 608060405260043610610062576000357c0100000000000000...
[ ] Bytecode length    : 7810
[ ] Blockchain contract: True
[ ] Debug              : False

[92m[-] The code does not contain SUICIDE instructions, hence it is not vulnerable[0m
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,

====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m............................Exception: ('Connection aborted.', BadStatusLine("''",))
[92m Sent! [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 87, in deploy_contract
    MyGlobals.web3.personal.unlockAccount(etherbase, '1', 15000)
  File "/usr/local/lib/python2.7/dist-packages/web3/personal.py", line 93, in unlockAccount
    [account, passphrase, duration],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1e771c5a50>: Failed to establish a new connection: [Errno 111] Connection refused',))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', error(104, 'Connection reset by peer'))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m......................
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f67bdef4410>: Failed to establish a new connection: [Errno 111] Connection refused',))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m...................
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
[1m[ ] Sending Ether to contract 0x9e536236abf2288a7864c6a1afaa4cb98d464306  [0m............................................Exception: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa44762b410>: Failed to establish a new connection: [Errno 111] Connection refused',))
[92m Sent! [0m
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 87, in deploy_contract
    MyGlobals.web3.personal.unlockAccount(etherbase, '1', 15000)
  File "/usr/local/lib/python2.7/dist-packages/web3/personal.py", line 93, in unlockAccount
    [account, passphrase, duration],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8550): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa4475f3390>: Failed to establish a new connection: [Errno 111] Connection refused',))
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
/usr/local/lib/python2.7/dist-packages/web3/main.py:130: DeprecationWarning: Python 2 support is ending! Please upgrade to Python 3 promptly. Support will end at the beginning of 2018.
  category=DeprecationWarning,
[1m[ ] Deploying contract [0m
====================================================================================================
[1m[ ] Compiling Solidity contract from the file /../data/reentrancy/solidity_contract/14274.sol ... [0m[92m Done [0m
[1m[ ] Connecting to PRIVATE blockchain emptychain  [0m.[92m ESTABLISHED [0m
Traceback (most recent call last):
  File "maian.py", line 182, in <module>
    main(sys.argv[1:])
  File "maian.py", line 120, in main
    if args.soliditycode:   contract_address = deploy_contract(args.soliditycode[1], MyGlobals.etherbase_account)
  File "/MAIAN/tool/contracts.py", line 99, in deploy_contract
    s.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/MAIAN/tool/contracts.py", line 115, in confirm_contract
    receipt = MyGlobals.web3.eth.getTransactionReceipt(transaction_creation_hash)
  File "/usr/local/lib/python2.7/dist-packages/web3/eth.py", line 192, in getTransactionReceipt
    [transaction_hash],
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 93, in request_blocking
    response = self._make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/manager.py", line 76, in _make_request
    return request_func(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/attrdict.py", line 20, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/middleware/formatting.py", line 25, in middleware
    response = make_request(method, params)
  File "/usr/local/lib/python2.7/dist-packages/web3/providers/rpc.py", line 52, in make_request
    **self.get_request_kwargs()
  File "/usr/local/lib/python2.7/dist-packages/web3/utils/compat/compat_requests.py", line 21, in make_post_request
    response = session.post(endpoint_uri, data=data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', error(104, 'Connection reset by peer'))
[1m[ ] Deploying contract [0m................